cmd/bosun/conf/conf.go

```diff
@@ -432,14 +441,23 @@ func (c *Conf) loadGlobal(p *parse.PairNode) {
 		c.PingDuration = d
 	case "noSleep":
 		c.NoSleep = true
-	case "putIPs":
+	case "BlockedPutIPs":
```

Comment:
I dislike the duplication in these two cases. Can we share a bit more of this logic please?